api/sendMail.js

```diff
@@ -0,0 +1,107 @@
+const nodemailer = require('nodemailer');
+const sanitizer = require('sanitize-html');
+require('dotenv').config();
+
+const from = `aleksey - ${process.env.EMAIL_ADDRESS}`;
+
+async function formSubmit(formData) {
+  const data = new Date();
+  return sendMail({
+    from: from,
+    to: 'alekseyhonchar@gmail.com',
+    subject: 'New user',
+    html: sanitizer(
+      `<ul><li>${formData.email}</li><li>${formData.name}</li></ul><br>${data}`
+    ),
+  });
+}
+
+const history = new Map();
```

Comment:
please reffer to article from chat - vars in functions scope are flushed on each run